src/pmlogger/pmlogger_farm_check.timer

```diff
@@ -2,10 +2,10 @@
 Description=5 minute check of pmlogger farm instances
 
 [Timer]
-# if enabled, runs 1m after boot and every 5 mins
+# if enabled, runs 1m after boot and every 2 mins
 OnBootSec=1min
-OnCalendar=*:00/5
+OnCalendar=*:00/2
```

Comment:
As discussed, there's no rationale been presented for this change - why 2 minutes now?  Why was it changed to 5 minutes from the original half-hourly a few weeks back?  And if we change it to be like this, surely the primary pmlogger would benefit from being on an increased checking interval as well?